system/include/AL/alext.h

```diff
@@ -42,436 +11,54 @@ typedef unsigned __int64 uint64_t;
 extern "C" {
 #endif
```

Comment:
Do we need the extern "C" anymore since we are not longer declaring any functions?

Reply:
It's probably a good idea to leave that in, in case anyone wants to bring in any extensions functions and they don't remember to add it back. It's found in the other headers, also, so it's uniform.